package.json

```diff
@@ -36,7 +36,7 @@
     "moment": "^2.20.1",
     "prop-types": "^15.6.0",
     "ramda": "^0.24.1",
-    "rc-slider": "^8.3.1",
+    "rc-slider": "^8.6.11",
```

Comment:
Necessary minor upgrade that adds the `tipProps` API (undocumented; last update to the changelog / [HISTORY.MD](https://github.com/schrodinger/rc-slider/blob/master/HISTORY.md) is from 2016).

Reply:
Seems like its repo has moved to https://github.com/react-component/slider - `HISTORY.MD` is still pretty sparse there, but `tipProps` appears to have been added back in 8.1.something. Anyway, I'm happy to have this up to date!